styles.css

```diff
@@ -0,0 +1,127 @@
+* {
+  box-sizing: border-box;
+  list-style: none;
+  font-family: "Roboto", "Crete Round", "Helvetica Neue", sans-serif, serif;
+}
+
+html {
+  overflow-x: hidden;
+}
+
+header {
+  width: 100vh;
+}
+
+body {
+  overflow-x: hidden;
+  overflow-y: auto;
+  margin: 0;
+  box-sizing: content-box;
+  background: rgba(255, 255, 255, 0.24);
+}
+
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100vw;
+  height: 92px;
+  background-color: #3c3a39;
+  position: fixed;
+  z-index: 100;
```

Comment:
- [ ] Great job 😃at implementing the header and nice design, but I think you put the height here as ```92 px```, but I see the height of the header is bigger than the design💯 I think according to the Figma Design it will be mostly ```70 px``` this will make the design beautiful,so I think you put the height of the charge😊 at design ,so please neglect the height of it I know that you tahe the height of 92 px with the charge of mobile up ,but kindly neglect the height of the charge icon👌 take a look below to see what I mean : 

| Figma design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/211883687-48d79f34-c085-468a-9a36-2305a65164e6.png)  | ![image](https://user-images.githubusercontent.com/83360636/211883983-8e388738-ff4b-454a-a08e-5c51cb962634.png) |